src/utils/deepLinks.ts

```diff
@@ -33,4 +33,5 @@ export const buildDeepLinkingMap = (org: Organization) => ({
   '/me/usage': `/orgs/${org.id}/usage`,
   '/me/users': `/orgs/${org.id}/users`,
   '/me/variables': `/orgs/${org.id}/settings/variables`,
+  '/user/profile': `/orgs/${org.id}/user/profile`,
```

Comment:
- this should be `/me/profile` not `/user/profile`
 - can you alphabetize this please?
 - like we mentioned offline, be sure to add a case for this new link to the e2e test
 - don't forget to add it to the deep link internal docs page either

Reply:
Is there a reason we want this to be `me` instead of `/user`? I see that's where the other deep links go, but wondered whether we should go with something different here since we're moving away from the concept of `me` elsewhere.